dotCMS/src/main/java/com/dotcms/integritycheckers/HostIntegrityChecker.java

```diff
@@ -325,13 +351,19 @@ private void fixHostConflicts(final Map<String, Object> row,
         // name. We need to have a dummy asset name because there is a
         // constraint that limit us to use the final one
         if (results == null || results.isEmpty()) {
-            final String temporalAssetName = "TEMP_" + UUID.randomUUID();
-            dc.setSQL("INSERT INTO identifier (id, parent_path, asset_name, host_inode, asset_type, syspublish_date, sysexpire_date)" +
-                    " SELECT ?, parent_path, '" + temporalAssetName + "', host_inode, asset_type, syspublish_date, sysexpire_date" +
-                    " FROM identifier WHERE id = ?");
-            dc.addParam(remoteHostIdentifier);
-            dc.addParam(localHostIdentifier);
-            dc.loadResult();
+            Logger.debug(
+                    this,
```

Comment:
use the supplier log

Logger.debug(this, ()-> "Hi Viko")

Reply:
Done.